common/string_calls.h

```diff
@@ -80,6 +80,8 @@ g_bool2text(int value);
 int
 g_text2bool(const char *s);
 
+char    *g_strnjoin(char *dest, int dest_len, const char *joiner, const char *src[], int src_len);
```

Comment:
Suggest Doxygen header for new funcs in this module.

Reply:
done